main/src/ui/conversation_titlebar/view.vala

```diff
@@ -54,14 +60,10 @@ public class ConversationTitlebar : Gtk.HeaderBar {
         set_title(Util.get_conversation_display_name(stream_interactor, conversation));
     }
 
-    private void update_subtitle(string? subtitle = null) {
-        if (subtitle != null) {
-            set_subtitle(subtitle);
-        } else if (conversation.type_ == Conversation.Type.GROUPCHAT) {
-            string subject = stream_interactor.get_module(MucManager.IDENTITY).get_groupchat_subject(conversation.counterpart, conversation.account);
-            set_subtitle(subject != "" ? subject : null);
-        } else {
-            set_subtitle(null);
```

Comment:
What's the reason for changing this function? With this change, if one switches from a MUC to a one-to-one chat, the one-to-one chat will still have the MUC topic set as a subtitle, because it's not cleared anymore.

Reply:
My bad. I've fixed this issue in 7ad6011 and also implemented a fallback to the groupchat's subject/jid's localpart (which is what Conversation does).